src/core/dbus-execute.c

```diff
@@ -1623,6 +1624,25 @@ int bus_exec_context_set_transient_property(
                                 if (len != 0)
                                         str[len - 1] = '\0';
 
+                                if (!c->cpuset) {
+                                        c->cpuset_ncpus = ncpus;
+                                        c->cpuset = (cpu_set_t*) memdup(a, sizeof(cpu_set_t) * n);
+                                } else if (c->cpuset_ncpus < ncpus) {
+                                        _cleanup_cpu_free_ cpu_set_t *cpuset;
+
+                                        cpuset = (cpu_set_t*) memdup(a, sizeof(cpu_set_t) * n);
+                                        if (!cpuset)
+                                                return -ENOMEM;
+
+                                        CPU_OR_S(CPU_ALLOC_SIZE(c->cpuset_ncpus), cpuset, c->cpuset, cpuset);
+
+                                        (void) mfree(c->cpuset);
```

Comment:
`(void) mfree()` is just plain `free()`… Please just use plain `free()` if you don't care about the return value... After all `mfree()` exists purely as syntactic sugar to permit `p = mfree(p)`...